tests/queries/0_stateless/02481_optimize_grouping_sets_keys.sql

```diff
@@ -0,0 +1,17 @@
+EXPLAIN QUERY TREE run_passes=1
```

Comment:
Need to rename test into `02481_analyzer_optimize_grouping_sets_keys.sql`, that way it is easy to run all analyzer tests.

Reply:
For this test we want to test `GROUP BY`, we do not need to add `HAVING`, `ORDER BY`, complex projection. Otherwise correctness of tests are hard to verify.

Reply:
I renamed the test, but see no point in editing it because I just copied queries from tests with the old analyzer a modified them a little bit.